static/app/components/replays/diff/replayDiffChooser.tsx

```diff
@@ -29,8 +30,12 @@ export default function ReplayDiffChooser({
 }: Props) {
   const isSameTimestamp = leftOffsetMs === rightOffsetMs;
 
+  const onTabChange = (tabKey: DiffType) => {
+    trackAnalytics('replay.hydration-modal.tab-change', {tabKey, organization: null});
+  };
```

Comment:
it's important to have the organization for slicing and dicing the data later, and it's easy to get a reference
`import useOrganization from 'sentry/utils/useOrganization';`
then:

```suggestion
  const organization = useOrganization();
  const onTabChange = (tabKey: DiffType) => {
    trackAnalytics('replay.hydration-modal.tab-change', {tabKey, organization});
  };
```